projects/ui-policy-studio-angular/src/lib/policy-studio/gio-policy-studio.apim-llm-proxy.stories.ts

```diff
@@ -0,0 +1,127 @@
+import { ApiType, Policy, SaveOutput } from '../models';
+import { fakePolicyDocumentation, fakePolicySchema } from '../models/policy/PolicySchema.fixture';
+import { fakeAllSharedPolicyGroupPolicies } from '../models/policy/SharedPolicyGroupPolicy.fixture';
+
+import { GioPolicyStudioComponent } from './gio-policy-studio.component';
+
+export default {
+  title: 'Policy Studio / APIM - Policy Studio / LLM Proxy API',
+  decorators: [
+    moduleMetadata({
+      imports: [GioPolicyStudioComponent],
+    }),
+    applicationConfig({
+      providers: [
+        matIconRegisterProvider(POLICIES_V4_UNREGISTERED_ICON.map(policy => ({ id: policy.id, svg: policy.icon }))),
+        importProvidersFrom(GioFormJsonSchemaModule),
+        GioElService,
+        provideAppInitializer(() => {
+          inject(GioElService).promptCallback = () => of({ el: 'Hello world!' });
+        }),
+      ],
+    }),
+  ],
+
+  render: props => ({
+    template: `<div style="height:calc(100vh - 2rem);"><gio-policy-studio
+    [readOnly]="readOnly"
+    [loading]="loading"
+    [apiType]="apiType"
+    [flowExecution]="flowExecution"
+    [entrypointsInfo]="entrypointsInfo"
+    [endpointsInfo]="endpointsInfo"
+    [commonFlows]="commonFlows"
+    [selectedFlowIndexes]="selectedFlowIndexes"
+    [plans]="plans"
+    [policies]="policies"
+    [sharedPolicyGroupPolicies]="sharedPolicyGroupPolicies"
+    [policySchemaFetcher]="policySchemaFetcher"
+    [policyDocumentationFetcher]="policyDocumentationFetcher"
+    [trialUrl]="trialUrl"
+    (selectedFlowChanged)="onFlowSelectionChange($event)"
+    (save)="onSave($event)"
+    >
+    </gio-policy-studio></div>`,
+    props: {
+      ...props,
+      policies: fakeAllPolicies(),
+      sharedPolicyGroupPolicies: fakeAllSharedPolicyGroupPolicies(),
+      trialUrl: 'https://gravitee.io/self-hosted-trial',
+      // Simulate a get policy schema http fetcher.
+      policySchemaFetcher: (policy: Policy) => of(fakePolicySchema(policy.id)).pipe(delay(600)),
+      policyDocumentationFetcher: (policy: Policy) => of(fakePolicyDocumentation(policy.id)).pipe(delay(600)),
+      onFlowSelectionChange: (event: { planIndex: number; flowIndex: number }) => {
+        console.info('onFlowSelectionChange', event);
+        action('onFlowSelectionChange')(event);
+      },
+      onSave: (event: SaveOutput) => {
+        console.info('saveOutput', event);
+        action('saveOutput')(event);
+      },
+    },
+  }),
+} as Meta;
+
+export const LlmProxyKafka: StoryObj = {
+  name: 'LLM API',
+  args: {
+    apiType: 'LLM_PROXY' satisfies ApiType,
+    entrypointsInfo: [fakeLlmProxyEntrypoint()],
+    endpointsInfo: [fakeLlmProxyEndpoint()],
+    commonFlows: [
+      fakeLlmFlow(base => {
+        return {
+          ...base,
+          name: 'Common flow',
+          selectors: [
+            {
+              type: 'HTTP',
+              methods: ['tools/list', 'tools/call', 'notifications/tools/list_changed'],
```

Comment:
🤔 Maybe more GET, POST or stuff like this. this is MCP stuff not HTTP
and path 
example :
```
{
          type: 'HTTP',
          pathOperator: 'EQUALS',
          path: '/test-path',
          methods: ['POST'],
        }
```